straxen/plugins/veto_pulse_processing.py

```diff
@@ -50,6 +49,11 @@ class nVETOPulseProcessing(strax.Plugin):
     data_kind = 'records_nv'
     ends_with = '_nv'
 
+    def setup(self):
+
+        self.baseline_samples = straxen.get_correction_from_cmt(self.run_id,
+                                                           self.config['baseline_samples_nv'])
+
```

Comment:
Regarding the failed test, muVETOPulseProcessing is a child of nVETOPulsePorcessing which overwrite the setting for `baseline_samples_nv` with an integer. Hence you have to add [here](https://github.com/XENONnT/straxen/blob/8ae2886c5e68503775782ccea38ba25f9ae54cd6/straxen/plugins/veto_pulse_processing.py#L250) a setup function too which overwrites the setup of nVETOPulseProcessing. The code would simply be:

```
def setup(self):
    self.baseline_samples = self.config['baseline_samples_mv']
```

Reply:
thanks, will add that